systems/llnl-sierra/system.py

```diff
@@ -42,8 +70,9 @@ def initialize(self):
         super().initialize()
 
         self.scheduler = "lsf"
-        self.sys_cores_per_node = "44"
-        self.sys_gpus_per_node = "4"
```

Comment:
I noticed `llnl-sierra` and `riken-fugaku` do not assign these attributes using `id_to_resources` like the other systems. Is there a reason this can't be changed?

Reply:
It can be changed if it helps - does it, and if so, how?  We started with the way sierra is specified, and the struct appeared later when we started using the same system spec for multiple systems.  Maybe this is an opportunity for redesign - it would be nice to have a standard, easy to maintain, and not error prone mechanism.  Ideas?

Reply:
`llnl-sierra` and `riken-fugaku` are probably set like this because `sys_cores_per_node` and `sys_gpus_per_node` are the same for `lassen` and `sierra`, and `riken-fugaku` has only 1 system. So there isn't any technical difference, but the change I made makes them have the same structure as all of the other systems.